src/kinetics/InterfaceKinetics.cpp

```diff
@@ -499,12 +499,33 @@ void InterfaceKinetics::getDeltaSSGibbs(doublereal* deltaGSS)
     // of the solution.
     for (size_t n = 0; n < nPhases(); n++) {
         thermo(n).getStandardChemPotentials(m_mu0.data() + m_start[n]);
+        //thermo(n).getGibbs_RT_ref(m_grt.data() + m_start[n]);
     }
 
     // Use the stoichiometric manager to find deltaG for each reaction.
+    //getReactionDelta(m_mu0.data(), deltaGSS);
```

Comment:
Please delete the newly added commented line

Reply:
Done